packages/typeorm-common/src/custom-repository/custom-repository.decorator.ts

```diff
@@ -0,0 +1,13 @@
+import { SetMetadata } from '@nestjs/common';
+
+export const REGISTER_CUSTOM_REPOSITORY_METADATA = 'CUSTOM_REPOSITORY_METADATA';
```

Comment:
acho mais safe escolher uma string mais única tipo

```suggestion
export const REGISTER_CUSTOM_REPOSITORY_METADATA = '@codemeistre/typeorm-common:CUSTOM_REPOSITORY_METADATA';
```

vai que alguma outra parte use o mesmo nome xD

Reply:
da p usar Symbol tb, n?

Reply:
dá mas symbol tem seu custo de memoria e processamento; pra esse caso, é mais facil e barato usar uma literal q a gente sabe q com ctz será unica